fundamentals-recap/practice-fundamentals-vocabulary.md

```diff
@@ -0,0 +1,14 @@
+# Practice: Fundamentals Vocabulary
+
+## Directions
+
+Complete these questions. Retry questions until you have the correct answer on each. Use any resource to answer these questions (including notes, other code, the Internet, assistance from others).
+
+When you finish, write down any remaining questions you have in your notes, and bring them to class.
+
+# Practice!
+
+<!-- Questions about vocab definitions -->
+
+<!-- Questions where they write code.... Need to figure out how ot make this testable -->
+<!-- Coding questions can include using type(), formatting strings, making list or dictionary literals -->
```

Comment:
```suggestion
<!-- Questions about vocab definitions -->

<!-- Questions where they write code.... Need to figure out how ot make this testable -->
<!-- Coding questions can include using type(), formatting strings, making list or dictionary literals -->

What is a statement?
a) a unit of code that can be evaluated into one value
b) a unit of code that performs an action
c) a value

What is the expression in this code snippet?
If score > 150000:
   print(“New High Score!”)

This code snippet below returns a `TypeError: can't multiply sequence by non-int of type 'str’`. It looks like we can’t multiply two strings. Just to double check, let’s check the data type of n1 and n2. Which option should we use to check the data type of n1?

```python
1  n1 = input("enter a number: ")
2 n2 = input("enter another number: ")
3
4 def multiply(n1, n2):
5  	return n1*n2
6
7 multiply(n1,n2)
``` 
a) type = n1
b) type(n1)
c) type.n1()
d) n1 = type()

This code snippet below returns a TypeError. Which lines could casting be implemented to prevent this error?

```python
1  n1 = input("enter a number: ")
2 n2 = input("enter another number: ")
3
4 def multiply(n1, n2):
5  	return n1*n2
6
7 multiply(n1,n2)

```
a) line 1
b) line 2
c) line 5
d) line 7


Which of the following options is a dictionary literal?

a) fruits = {
        "apple": "gala apple",  
        "orange": "sumo orange", 
        "pear": "anjou pear",
       }
b) fruits
c) {
        "apple": "gala apple",  
        "orange": "sumo orange", 
        "pear": "anjou pear",
       }
d) "orange": "sumo orange"
```

adding Learn markdown tomorrow morning